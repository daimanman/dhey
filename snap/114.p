{"C":1,"Duration":131660168,"EndTime":1551409761141214600,"Err":"","FileId":"","InputFileName":"","LonhApi":"saveFiregroundpm","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409761009554400,"Status":1,"TaskId":114,"Type":"LONH","Url":"","Z":""}