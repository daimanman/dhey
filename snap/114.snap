{"AvgTotal":0.1136642,"Fastest":0.1136642,"Slowest":0.1136642,"Average":0.1136642,"Rps":8.797844879918214,"AvgConn":0.0548769,"AvgDNS":0.0069812,"AvgReq":0,"AvgRes":0.000985,"AvgDelay":0.056837,"ConnMax":0.0069812,"ConnMin":0.0069812,"DnsMax":0.0069812,"DnsMin":0.0069812,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.000985,"DelayMax":0.056837,"DelayMin":0.056837,"Lats":[0.1136642],"ConnLats":[0.0069812],"DnsLats":[0.0069812],"ReqLats":[0],"ResLats":[0.000985],"DelayLats":[0.056837],"Total":113664200,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1136642,"Count":1,"Frequentcy":1},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0},{"Mark":0.1136642,"Count":0,"Frequentcy":0}],"TaskId":114,"Duration":0.1136642}